docs/pages/interfaces.rst

```diff
@@ -564,6 +564,15 @@ IOResult specific
   :members:
   :private-members:
 
+Failable specifc
```

Comment:
I think that `Failable` is not specific.

We can interfaces `specific` when they do work with some our existing types.
`Failable` does not work with any of them.

Reply:
Done!